stackable-base/Dockerfile

```diff
@@ -140,6 +142,9 @@ echo -e "if [ -f ~/.bashrc ]; then\n\tsource ~/.bashrc\nfi" >> /stackable/.profi
 chown stackable:stackable /stackable/.bashrc
 chown stackable:stackable /stackable/.profile
 
+cp /root/.curlrc /stackable/.curlrc
+chown stackable:0 /stackable/.curlrc
```

Comment:
Nice, was going to mention doing it for any users we switch to (so deleting my previous comment).